examples/md-flexible/input/AllOptions.yaml

```
@@ -16,7 +16,7 @@ verlet-rebuild-frequency             :  20
 verlet-skin-radius-per-timestep      :  0.01
 verlet-cluster-size                  :  4
 selector-strategy                    :  Fastest-Absolute-Value
-tuning-metric                        :  time
+tuning-metric                        :  time, energy, energyPerFLOP, energyDelayProduct
```

```suggestion
tuning-metric                        :  time #alternatively: energy, energyPerFLOP, energyDelayProduct
```
Show other available options as a comment, as only one can be chosen at a time.